source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,337 @@ Sorts Builders
+
+If you need guaranteed sort order for documents that
+have fields with identical values, you can specify additional fields to sort
+on in the event of a tie.
+
+We can specify an ascending sort on the ``letter`` field followed by the
+``_id`` field as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"));
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+Combining Sort Criteria
+-----------------------
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method builds sort criteria that apply passed 
+in sort criteria from left to right in the event of ties. 
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
+descending sort on the ``letter`` field with an ascending sort on the
+``_id`` field.
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.orderBy;
+   import static com.mongodb.client.model.Sorts.ascending;
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+
+   Bson orderBySort = orderBy(descending("letter"), ascending("_id"));
+   collection.find().sort(orderBySort);
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+
+Text Search
+-----------
+
+You can specify the order of the results of a text search performed with the
+:manual:`$text </reference/operator/query/text/>` query operator
+by how closely they match your search string. Each of your search results has a
+:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>`, a
+numerical value indicating how well that result matches your search. 
+Use the ``Sorts.metaTextScore()`` static factory method to build your sort
+criteria to sort by the text score.
+
+.. warning:: Make Sure to Create a Text Index
+
+   Text searches require a :manual:`text index </core/index-text/>` to locate
+   and score your results. See the server manual documentation for more
```

Comment:
Nitpick: I think you can make this more concise and precise in that the collection you're querying requires it. Otherwise you might assume an index is something you can pass to a text search.

```suggestion
   You need a :manual:`text index </core/index-text/>` on your collection to perform a text search. See the server manual documentation for more
```

Reply:
Thank you for the suggestion. On rereading, I agree this is a more precise phrasing. I will try to keep an eye out for multiple meanings in my sentences going forward.